File: 백준/Bronze/5598. 카이사르 암호/카이사르 암호.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String word = br.readLine().trim();
        StringBuilder result = new StringBuilder();
        
        for (char wo : word.toCharArray()) {
            int add = wo - 3;
            
            if (add < 'A') {
                add += ('Z' - 'A' + 1);
            }
            result.append((char) add);
        }
        
        System.out.println(result.toString());
    }
}

File: 백준/Bronze/5598. 카이사르 암호/카이사르 암호.py
import sys

input = sys.stdin.readline

word = input().strip()
result = ""

for wo in word:
    add = ord(wo) - 3
    if ord(wo) - 3 < ord('A'):
        add += (ord('Z') - ord('A') + 1)
    result += chr(add)

print(result)